modules/administration-guide/partials/proc_configuring-che-with-custom-resources-to-use-amazon-s3.adoc

```diff
@@ -0,0 +1,62 @@
+[id="configuring-{prod-id-short}-with-custom-resources-to-use-amazon-s3_{context}"]
+= Configuring {prod-short} with custom resources to use Amazon S3
+
+To configure {prod-short} to use Amazon S3:
+
+.Prerequisites
+
+* Configured Amazon S3. See xref:supported-restic-compatible-backup-servers.adoc[].
+
+.Procedure
+
+. Create a secret containing the `repo-password` key with a Base64 encoded password:
++
+[source,yaml,subs="+quotes,+attributes"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: backup-encryption-password-secret
+  namespace: eclipse-che
+type: Opaque
+stringData:
+  repo-password: __<base64_encoded_password>__
+----
++
+({prod-short} Operator will set up this password for the backup repository that {prod-short} Operator will create from this custom object on the backup server.)
++
+include::snip_warning-about-losing-backup-repo-password.adoc[]
+
+. Create a secret (for example, `name: aws-user-credentials-secret`) containing:
+.. `awsAccessKeyId` key with the Base64 encoded AWS access key ID of the user
```

Comment:
Item should be bulleted, and not a sub-step

Reply:
Thank you. Fixed.